keel-core/src/main/kotlin/com/netflix/spinnaker/keel/verification/VerificationRunner.kt

```diff
@@ -0,0 +1,81 @@
+package com.netflix.spinnaker.keel.verification
+
+import com.netflix.spinnaker.keel.api.Verification
+import com.netflix.spinnaker.keel.api.plugins.VerificationEvaluator
+import com.netflix.spinnaker.keel.api.verification.VerificationContext
+import com.netflix.spinnaker.keel.api.verification.VerificationRepository
+import com.netflix.spinnaker.keel.api.verification.VerificationStatus
+import com.netflix.spinnaker.keel.api.verification.VerificationStatus.RUNNING
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+
+@Component
+class VerificationRunner(
+  private val verificationRepository: VerificationRepository,
+  private val evaluators: List<VerificationEvaluator<*>>
+) {
+  fun runVerificationsFor(context: VerificationContext) {
```

Comment:
Really nice use of extension functions in this method to abstract details and help the code tell a story. 👏